src/seeders/functionalities/HeaterActuatorSeeder.js

```diff
@@ -10,12 +10,16 @@ class HeaterActuatorSeeder extends ActuatorSeeder {
     ]
   }
 
-  static generate(data) {
-    return {
-      ...super.generate(data),
+  static generate(overrideData = {}) {
+    const fctData = {
+      ...super.generate(overrideData),
       subType: HeaterActuator.SUB_TYPE,
-      slots: [ ...this.SLOT_SEEDS ],
-      ...data,
+    }
+
+    return {
+      ...fctData,
+      slots: [ ...this.generateSlots(overrideData.id || fctData.id) ],
+      ...overrideData,
```

Comment:
then use the id when making the slots (or use the provided id)

(the changes to this FCT seeder are in all of the usable FCT seeders)